_scripts/webpack.renderer.config.js

```diff
@@ -146,4 +147,14 @@ const config = {
   target: 'electron-renderer',
 }
 
+if (isDevMode) {
+  config.plugins.push(
+    new WatchExternalFilesPlugin({
+      files: [
+        './static/locales/*.yaml',
```

Comment:
As only 45 out of the 65 locale files are actually enabled, it might make more sense to read the list from the activeLocales.json file, instead of having it watch all of them. (you'll need to read the activeLocales.json file too, but here is how you could tell it to only watch the necessary files.
```suggestion
        `./static/locales/{${activeLocales.join(',')}}.yaml`,
```